specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -623,7 +555,9 @@
    "metadata": {},
    "source": [
     "### 4f) Visualize the results\n",
-    "Using the python package *hvplot*, we can easily make a line plot of the annual aggregated data. To do this, we'll plot the annual HDD, then add the trendline on top. The code to generate the plot is contained in a function `hdd_cdd_lineplot`. "
+    "Using the python package *hvplot*, we can easily make a line plot of the annual aggregated data. To do this, we'll plot the annual HDD, then add the trendline on top. The code to generate the plot is contained in a function `hdd_cdd_lineplot`. \n",
```

Comment:
hdd lineplot just looks like a flat line... 
![hdd_lineplot](https://user-images.githubusercontent.com/11436996/229931783-d9a3bbbf-37e2-4178-918e-68f948bc8d11.png)

Reply:
@thenaomig I can't reproduce this and do get the plots to run for me

Reply:
Okay, it works for me now too.